index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```

Comment:
```suggestion
    <meta charset="UTF-8" viewport="width=device-width, initial-scale=1.0">
```
add viewport attribute to the meta tag
this is necessary to Control the size of the viewport.